src/pipelines/epidemiology/config.yaml

```diff
@@ -404,6 +404,7 @@ sources:
       # TODO(owahltinez): skip test since it's very flakey
       skip: true
       metadata_query: "key == 'GB'"
+    backfill: true
```

Comment:
Would it make more sense to make `backfill` an option under `parse` rather than at the root of the data source config?

Reply:
Done.